build.gradle.kts

```diff
@@ -21,9 +22,14 @@ version = properties("pluginVersion").get()
 // Configure project's dependencies
 repositories {
     mavenCentral()
+//    maven { url = uri("https://repository.jboss.org/nexus/content/repositories/snapshots") }
```

Comment:
I think you dont need that. Lsp4mp is the MicroProfile language server that we use in IJ Quarkus

Reply:
Thank you for your comment.
I merged the main and added some steps to build it. I copied it from IJ Quarkus as a reference.